src/Parser/Mage/Frost/CombatLogParser.js

```diff
@@ -7,6 +7,8 @@ import Talents from 'Main/Talents';
 import CoreCombatLogParser from 'Parser/Core/CombatLogParser';
 import DamageDone from 'Parser/Core/Modules/DamageDone';
 
+import ShardOfTheExodar from 'Parser/Mage/Shared/Modules/Items/ShardOfTheExodar';
```

Comment:
It's not a huge deal, mainly depends on if you want to use relative paths or absolute paths. The import path should be equal to (you start in Frost) `../Shared/Modules/Items/ShardOfTheExodar`

Reply:
I changed this back to a relative path since its only one directory back and because i wanted to keep all the items together.